content/docs/alerting/configuration.md

```diff
@@ -64,12 +64,17 @@ global:
   # The default SMTP From header field.
   [ smtp_from: <tmpl_string> ]
   # The default SMTP smarthost used for sending emails.
+  # Provide the correct port for the smarthost (25 / 587 for SMTP over TLS (STARTTLS), 465 for deprecated SMTPS). 
+  # Example: smtp.example.org:587
   [ smtp_smarthost: <string> ]
-  # SMTP authentication information.
+  # Set the username if required by the server.
   [ smtp_auth_username: <string> ]
+  # SMTP authentication information for SMTP Auth using LOGIN.
```

Comment:
This is incorrect, it's also used by PLAIN.

Any documentation here needs to be 100% correct and precise as it is reference documentation, not just what works in some setups.